build.gradle

```diff
@@ -4,8 +4,8 @@ plugins {
 }
 
 ext.versions = [
-  'kotlin': gradle.ext.isKotlinDev ? '1.4.0-rc' : '1.3.70',
-  'gradle': '6.5'
+  'kotlin': gradle.ext.isKotlinDev ? '1.4.0-rc' : '1.4.0',
```

Comment:
Since there's one stable release. Does it make sense keeping this release candidate here?